dev/partkey.md

```diff
@@ -0,0 +1,163 @@
+---
+numbersections: true
+title: "Algorand Key Specification"
+date: \today
+abstract: >
+    This document specifies the list of keys and their capabilities in Algorand.
+---
+
+Overview
+========
+
+An algorand node uses two kind of cryptographic keys:
+
+ - _participation keys_, a set of keys used for authentication, i.e. identify an 
+    account. Algorand uses a two\-level ephemeral signature scheme that 
+    ensures [forward security](https://en.wikipedia.org/wiki/Forward_secrecy), 
```

Comment:
This is the wrong wikipedia article; forward _secrecy_ is a property of key exchange / encrypted messaging protocols, not signature schemes

Reply:
This wikipedia article indeed does not talk about forward-secure signatures. One solution is to cite https://cseweb.ucsd.edu/~mihir/papers/fsig.pdf (which is the first paper to formalize forward-secure signatures). What do you think @shaih?